ivy/functional/frontends/numpy/random/functions.py

```diff
@@ -175,3 +175,13 @@ def rayleigh(scale, size=None):
     log_u = ivy.log(u)
     x = ivy.multiply(scale, ivy.sqrt(ivy.multiply(-2, log_u)))
     return x
+
+@to_ivy_arrays_and_back
+@from_zero_dim_arrays_to_scalar
+def permuted(x, axes=None, out=None):#, copy=None):
+    if out != x:  #when out is not x or is None, shuffle in the result
```

Comment:
`out`'s should not be checked against anything. If out is not None, just need to store result in out and return it